src/newmatrix.cpp

```diff
@@ -242,19 +242,21 @@ void roll_matrix_init(struct matrix_icon *icon)
   int init_dice = 1;
   if (icon->decker && icon->decker->ch)
   {
-    // Matrix pg 18 & 24, available bonuses are response increase, reality filter, and hot asist
-    init_dice += GET_INIT_DICE(icon->decker->ch) + icon->decker->response + (icon->decker->reality ? 1 : 0) + (icon->decker->asist[0] ? 1 : 0);
-
     if (icon->type == ICON_LIVING_PERSONA) {
-      init_dice = MIN(5, init_dice + GET_ECHO(icon->decker->ch, ECHO_OVERCLOCK));
+      // Matrix pg 138 & 145, 4 dice with a possible +1 from overclock
+      init_dice = 4 + (GET_ECHO(icon->decker->ch, ECHO_OVERCLOCK) ? 1 : 0);
+      // This is the living persona's matrix reaction, not response increase
+      icon->initiative = icon->decker->response;
```

Comment:
Response is a flat value here, but doubled for non-otaku. Is that intended?

Reply:
Yes, this is intended. For a deck, each level of response gives +2 reaction and +1 init die, and is limited to 0-3. The code that sets `GET_CYBERDECK_RESPONSE_INCREASE` (which then gets passed to `DECKER->response` in `do_connect`) in `update_living_persona` is actually calculating the otaku's matrix reaction, not level of response increase. Otaku don't actually get response increase levels, so I'm letting the two different things use the same struct member.

For clarity, should I get rid of the `GET_CYBERDECK_RESPONSE_INCREASE` calculation in `update_living_persona` and instead calculate the otaku's matrix reaction here?

Reply:
It'd probably be clearer, but I'm okay with it as-is. Thanks!